shell/nuxt.config.js

```diff
@@ -27,6 +27,8 @@ const version = process.env.VERSION ||
   process.env.DRONE_VERSION ||
   require('./package.json').version;
 
+const dashboardVersion = process.env.DASHBOARD_VERSION;
```

Comment:
Same concept here, we need to ensure people understand the difference between version and dashboard version. I think though we could be brave and remove `version` and `commit` from here.

Reply:
Agree, they can both go